concordia/static/scss/action-app.scss

```diff
@@ -245,15 +376,17 @@ main {
 }
 
 #help-panel {
-    flex-basis: 26rem;
-    ul {
-        padding-left: 1rem;
+    overflow: auto;
+    padding: 1rem;
+    flex-basis: 285px;
+    min-width: 285px;
+    * {
+        font-size: $concordia-app-font-size-xs;
     }
 }
 
-.control-toolbar {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-around;
-    align-items: center;
+.concordia-app-control-toolbar {
```

Comment:
This should have a name which better conveys what it applies to — e.g. the asset list, the image viewer, etc. — since this app is going to expand functionality over time